### ISSUE TITLE
Profile viewer has an extremely limited color palette

### ISSUE THREAD

**Issue by LoganDark** [plasma-umass/coz#196]
The profile viewer only has 4 colors so it's impossible to tell progress points apart if you have more than 4.

The tooltip also does not display which progress point a line is.

https://user-images.githubusercontent.com/4723091/173958013-99f90fcf-931f-47ed-9b17-52c744194230.mp4

**Comment by fenollp** [comment]
+1 for colorblind people :)

**Comment by LoganDark** [comment]
> +1 for colorblind people :)

Yeah, the tooltip really needs to display where the line comes from. I guess the extremely limited color palette simulates color blindness. I'm artificially limited in what I can tell apart. :(
